src/plugins/Panel/index.ts

```diff
@@ -12,6 +12,14 @@ type PanelQueueItem = {
 
 export const queue: PanelQueueItem[] = reactive([])
 
+export function close(): void {
```

Comment:
we should probably give these more precise names like `closePanel` and `exitPanels`. Not only to match `showPanel` but to prevent against future collisions when importing from `@prefecthq/miter-design`

Reply:
agreed, sorry I didn't see your comment prior. Creating a follow up PR for this.

Reply:
https://github.com/PrefectHQ/miter-design/pull/209